src/api/app/controllers/webui/watched_items_controller.rb

```diff
@@ -20,7 +21,10 @@ def toggle_watched_item
       flash[:success] = "Added #{FLASH_PER_WATCHABLE_TYPE[@item.class]} to the watchlist"
     end
 
-    redirect_back(fallback_location: root_path)
+    respond_to do |format|
+      format.html
```

Comment:
Why keep that around?

Reply:
We don't need it. All the requests to this route should be js requests now. Thanks for pointing that out.